### ISSUE TITLE
Include LCM samplers

### ISSUE THREAD

**Issue by GeminiSquishGames** [benrugg/AI-Render#145]
### Describe the feature you'd like to see:

I use LCM samplers and checkpoints for fast generation, especially useful for motion, but I can't seem to get them to appear in the plugin UI. It does show up in sdapi/v1/samplers, and most other plugins or programs that utilize API. Would be nice to have.

**Comment by benrugg** [comment]
Yes, I'd like to add support for this. It's similar to this issue: https://github.com/benrugg/AI-Render/issues/137

I don't have time to do this currently, but any collaboration would be appreciated!
